values.yaml

```diff
@@ -25,7 +25,8 @@ agent:
   podAnnotations: {}
 
   # -- Security Context policies for agent pods
-  podSecurityContext: {}
+  podSecurityContext:
```

Comment:
I will move this to a template when cleaning this PR up. It's needed to allow the `circleci` user in the agent container write permissions to the mounted volume